pastis/optimization/setup.py

```diff
@@ -3,7 +3,7 @@
 def configuration(parent_package='', top_path=None):
     from numpy.distutils.misc_util import Configuration
 
-    config = Configuration('optimization', parent_package, top_path)
+    config = Configuration('inference', parent_package, top_path)
```

Comment:
Whoops, that is totally my bad! In topsy, the package was originally called "inference" rather than "optimization". I actually ended up changing this yesterday so that it would match pastis, but hadn't committed it yet. I'll commit the fix today.